src/app/course/practice-problem/practice-problem.component.html

```diff
@@ -92,73 +65,125 @@
+                    </div>
+                </div>
+            </div>
+            <div class="tui-container tui-container_adaptive tui-container_example">
+                <div class="tui-row tui-row_adaptive">
+                    <div class="tui-col_xs-2 tui-col_md-2 tui-col_lg-2">
+                        <div class="dummy">
+                            <p class=tui-space_top-6>Include Solved Questions:</p>
+                        </div>
+                    </div>
+                    <div class="tui-col_xs-1 tui-col_md-1 tui-col_lg-1">
+                        <div class="dummy">
+                            <tui-checkbox-labeled
+                                (ngModelChange)="updateQuestions(difficulty, $event)"
+                                [class.tui-skeleton]="!difficulties || !category"
+                                [ngModel]="include_solved"
+                                class="tui-space_top-10"
+                                size="l"
+                                ngDefaultControl
+                            >
+                            </tui-checkbox-labeled>
+
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div>
+                <p class=tui-space_horizontal-10>Browse this category:</p>
+                <div class="tui-container tui-container_adaptive tui-container_example">
+                    <div class="tui-row tui-row_adaptive">
+                        <div class="tui-col_xs-1 tui-col_md-1 tui-col_lg-1">
+                            <div class="dummy">
+                                <a
+                                    class="tui-space_vertical-0 tui-space_horizontal-0"
+                                    (click)="nextQuestion()"
+                                    [class.tui-skeleton]="!category || !uqjs"
+                                    icon="tuiIconArrowLeft"
+                                    size="xl"
+                                    tuiButton
+                                >
+
+                                </a>
+
+                            </div>
+                        </div>
+
+                        <div class="tui-col_xs-1 tui-col_md-1 tui-col_lg-1">
+                            <div class="dummy">
+                                <a
+                                    class="tui-space_vertical-0 tui-space_horizontal-10"
+                                    (click)="nextQuestion()"
+                                    [class.tui-skeleton]="!category || !uqjs"
+                                    icon="tuiIconArrowRight"
+                                    size="xl"
+                                    tuiButton
+                                >
+
+                                </a>
+
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <br>
+            <br>
+            <br>
+            <br>
```

Comment:
Let's not have a bunch of `<br>`s. you should handle the spacing with class names.